environment-dev.yml

```diff
@@ -62,12 +62,12 @@ dependencies:
   - pandas-stubs>=2.0.0
 
   - pip:
+      - asv==0.5.1
```

Comment:
Let's leave the installation with pip for now, I vaguely remember some problems when installing via conda.